### PR TITLE
IR long press support.

### PR DESCRIPTION
**Pull request by exislow**
To be able to adjust, e.g. volume more convenient remote button long press is now supported. If you press, for instance, UP for a long time, the volume will rise until you release the button on the IR.

Also code refactoring for `aurora.ino` was done to be able to maintaine and read the code better.

### CODE REVIEW

File: SOURCES/WEBAPP/ESP32/aurora/hwconfig.h [hunk 1]
```diff
@@ -33,15 +33,16 @@
...
-#define APPLE_REMOTE_UP          0x77E1D047
-#define APPLE_REMOTE_DOWN        0x77E1B047
-#define APPLE_REMOTE_LEFT        0x77E11047
-#define APPLE_REMOTE_RIGHT       0x77E1E047
-#define APPLE_REMOTE_CENTER_DOWN 0x77E1BA47
-#define APPLE_REMOTE_CENTER_UP   0x77E12047
-#define APPLE_REMOTE_MENU        0x77E14047
-#define APPLE_REMOTE_PLAY_DOWN   0x77E17A47
-#define APPLE_REMOTE_PLAY_UP     0x77E12047
+#define APPLE_REMOTE_UP          0x77E1D055
```
Comment (zas) [inline SOURCES/WEBAPP/ESP32/aurora/hwconfig.h:36]: https://github.com/brackendawson/Appleceiver/blob/357a0a16013d9e35f1119530399aba55b7030bab/Appleceiver.ino#L43-L55 suggests it should be possible to support multiple Apple remotes using correct bit mask.

**Comment by zas** [comment]
@exislow you have to base your PR on develop branch I think

File: SOURCES/WEBAPP/ESP32/aurora/hwconfig.h (hunk 1 above)
Comment (exislow) [reply SOURCES/WEBAPP/ESP32/aurora/hwconfig.h:36]: Totally agree. But I would like to open another PR for this.

**Comment by dspverden** [comment]
Instead of changing the source code with logic expression I would rather prefer to see the codes read from a config file, this will enable other IR remotes then Apple only.

**Comment by zas** [comment]
> Instead of changing the source code with logic expressions I would rather prefer to see the codes read from a config file, this will enable other IR remotes then Apple only.

I open the path to that in https://github.com/freeDSP/freeDSP-aurora/pull/97

Basically read IR value is passed to a function which returns a code for the button pressed, we can extend it to match more remotes more easily.
It supports repeat too.
Please comment on the PR itself

**Comment by zas** [comment]
@exislow can you test if https://github.com/freeDSP/freeDSP-aurora/pull/97 works for you?

**Comment by zas** [comment]
@exislow if you're ok, I think we can close this PR, as https://github.com/freeDSP/freeDSP-aurora/pull/97 includes this feature.

**Comment by exislow** [comment]
Sure.

**Pull request closed without merging**